README.md

```diff
@@ -38,6 +37,30 @@ import {URLPattern} from "urlpattern-polyfill";
 globalThis.URLPattern = URLPattern
 ```
 
+## loading as CommonJs module
+
+```javascript
+// Conditional CJS module loading (Node.js)
+if (!globalThis.URLPattern) {
+   require("urlpattern-polyfill");
+}
+/** 
+ * The above is the recommended way to load the CommonJs module, as it only
+ * loads it on demand, thus when not natively supported by the runtime or
+ * already polyfilled.
+ */
+require("urlpattern-polyfill");
+
+/** 
+ * In case you want to replace an existing implementation with the polyfill:
+ */
+const {URLPattern} = require("urlpattern-polyfill");;
+globalThis.URLPattern = URLPattern
+```
+
+> ## Note:
+> No matter how you load the polyfill, when there is no implementation in your environment, it will _allways_ add it to the global.
```

Comment:
```suggestion
> No matter how you load the polyfill, when there is no implementation in your environment, it will _always_ add it to the global object.
```